<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="en" sourcelanguage="en">
<context>
    <name>DataController</name>
    <message>
        <location filename="../src/lib/datacontroller.cpp" line="80"/>
        <location filename="../src/lib/datacontroller.cpp" line="104"/>
        <source>Error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../src/lib/datacontroller.cpp" line="81"/>
        <source>The server reported an error.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../src/lib/datacontroller.cpp" line="93"/>
        <source>If the problem persists, please contact the developer.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../src/lib/datacontroller.cpp" line="100"/>
        <source>Network Error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../src/lib/datacontroller.cpp" line="101"/>
        <source>A network error occured</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../src/lib/datacontroller.cpp" line="105"/>
        <source>An unknown error occured, we don&apos;t known more</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
